tests/test_coverage_gaps.py

```
@@ -0,0 +1,1276 @@
+Tests for specific uncovered lines across ble.py, async_utils.py, events.py,
+and command_bridge.py.
+"""
+
+import asyncio
+import threading
+import time
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+import mmrelay.matrix_utils as matrix_utils
```

Ruff (0.15.10)

[warning] Use `from mmrelay import matrix_utils` in lieu of alias

Replace with `from mmrelay import matrix_utils`

(PLR0402)